test/cypress/integration/group-proposals.spec.js

```diff
@@ -2,6 +2,10 @@ const userId = Math.floor(Math.random() * 10000)
 const groupName = 'Dreamers'
 const groupMincome = 250
 const groupNewMincome = 500
+const groupInviteLinkExpiry = {
+  anyone: 30,
+  proposal: 7
+}
```

Comment:
So that they don't get out-of-sync, can you import these values from `frontend/model/contracts/constants.js`?

Reply:
Sure. This needs to be done.